apps/web-giddh/src/app/models/defaultMenus.ts

```diff
@@ -117,9 +117,9 @@ export let DEFAULT_MENUS_ORIGINAL: IUlist[] = [
     {
         type: 'MENU', name: 'Invoice', uniqueName: '/pages/invoice/preview/sales', isRemoved: false, pIndex: 2, additional: { tab: 'sales', tabIndex: 0 }
     },
-    {
-        type: 'MENU', name: 'Journal Voucher', uniqueName: '/pages/accounting-voucher', isRemoved: false, pIndex: 1
-    },
+    // {
```

Comment:
will this commented code reuse in future?

Reply:
It's JV code commented for some time as new JV module is under development.